### DISCUSSION TITLE
Widget support for homescreen to quickly glance at feeds

### DISCUSSION THREAD

**Discussion by shivamashtikar** [spacecowboy/Feeder#237] [Ideas]
Widget support for Home Screen to quickly glance at feeds 
Widgets for different use cases like
- widget for quick open categories
- widget which displays selected feeds directly in Home Screen

**Comment by spacecowboy** [comment]
Commonly requested. But I don't really use widgets so have no need to built it.

Contributions are welcome though.

See https://developer.android.com/jetpack/compose/glance

**Reply by shivamashtikar** [reply]
Sure @spacecowboy , Happy to contribute, will raise PR for widgets support. Will use this thread for communication
